SynthDet/Assets/Scripts/Randomizers/ForegroundOccluderScaleRandomizer.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using SynthDet.RandomizerTags;
+using UnityEngine;
+using UnityEngine.Perception.Randomization.Parameters;
+using UnityEngine.Perception.Randomization.Randomizers;
+using UnityEngine.Perception.Randomization.Samplers;
+
+namespace SynthDet.Randomizers
+{
+    [Serializable]
+    [AddRandomizerMenu("Perception/My Foreground Occluder Scale Randomizer")]
+    public class ForegroundOccluderScaleRandomizer : Randomizer
```

Comment:
Part of a bigger discussion around placement algorithms, why choose to randomize scale separately from placement of foreground occluders? It seems like it would save a fair bit of object setup and project complexity if you didn't have to have these tags.

Reply:
True, there is a level of optimization that can go into parts of the scenario design. In this case, it was purely an educational thing, so that the user could understand exactly what steps are happening.